google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/it/ITBigQueryStorageTest.java

```diff
@@ -104,6 +151,57 @@ public void testSimpleRead() {
     assertEquals(164_656, avroRowCount);
   }
 
+  @Test
+  public void testSimpleReadAndResume() {
+    TableReference tableReference =
+        TableReference.newBuilder()
+            .setProjectId("bigquery-public-data")
+            .setDatasetId("samples")
+            .setTableId("shakespeare")
+            .build();
+
+    ReadSession session =
+        client.createReadSession(
+            /* tableReference = */ tableReference,
+            /* parent = */ parentProjectId,
+            /* requestedStreams = */ 1);
+    assertEquals(
+        String.format(
+            "Did not receive expected number of streams for table reference '%s' CreateReadSession response:%n%s",
+            TextFormat.shortDebugString(tableReference), session.toString()),
+        1,
+        session.getStreamsCount());
+
+    // We have to read some number of rows in order to be able to resume. More details:
+    // https://cloud.google.com/bigquery/docs/reference/storage/rpc/google.cloud.bigquery.storage.v1beta1#google.cloud.bigquery.storage.v1beta1.ReadRowsRequest
+
+    long avroRowCount = SkipNumberOfRows(session.getStreams(0), /* rowsToSkip = */ 34_846);
```

Comment:
Naming of SkipNumberOfRows implied to me you're merely forwarding the offset, but it turns out it also performs reads to that point due to the need to allocated data to the stream.  Would this be better named something like ReadStreamToOffset?

Reply:
@shollyman , feedback applied in commit:
https://github.com/googleapis/google-cloud-java/pull/5242/commits/83a7a95a3e54b85cf83e7d7b10b2b293f81dd5fc